app/templates/people/person/personal-information/index.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading>
+            Persoonlijke gegevens
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup>
+          <AuLink
+            @route="people.person.personal-information.edit"
+            @skin="button-secondary"
+            @icon="pencil"
+            @iconAlignment="left"
+          >
+            Bewerk
+          </AuLink>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3">
+        {{@model.person.givenName}} {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Voornaam</:label>
+            <:content>{{@model.person.givenName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Achternaam</:label>
+            <:content>{{@model.person.familyName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          {{#if @model.person.firstNameUsed}}
+          <Item>
+            <:label>Roepnaam</:label>
+            <:content>{{@model.person.firstNameUsed}}</:content>
+          </Item>
+          {{/if}}
+        </Grid>
+
+      </:card>
+    </DataCard>
+
+    <DataCard>
+      <:title>Contactgegevens:
+        {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.classification.label}}
+      </:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Organisatie</:label>
+            <:content>
+              <AuLink
+                @model={{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.id}}
+                @route="administrative-units.administrative-unit">
+                {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.name}}
+              </AuLink>
+            </:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
```

Comment:
A lot of these contact thingies seem empty (for the people I checked at least), so we shouldn't show the label either in that case. I would wrap all of them in if checks.

Reply:
Ok but in that case to keep the layout I have to use different grids.